src/Models/class-inbound-smart-link.php

```diff
@@ -1069,9 +1069,8 @@ public static function get_suggestions_count( int $post_id ): int {
 		$cache_key = self::get_suggestions_count_cache_key( $post_id );
 		$count     = wp_cache_get( $cache_key, PARSELY_CACHE_GROUP );
 
-		if ( false !== $count ) {
-			/** @var int $count */
-			return $count;
+		if ( is_numeric( $count ) ) {
```

Comment:
Can we make all of these checks explicit for checking `false !==`? `wp_cache_get()` is explicitly going to return `false` when it fails, so it's a bit odd to check for typing in those cases.

```suggestion
		if ( false !== $count && is_numeric( $count ) ) {
```

Reply:
I've removed it as unnecessary (e.g. if it's numeric, it's not false) but I'll bring these back if you feel they're more readable.

Reply:
I'd prefer them in, as `wp_cache_get()` returning `false` has a specific meaning that isn't exactly the same as `is_numeric()`. I'd also prefer it for readability as well, even if it's verbose.

Reply:
OK, I'll bring these back! 🙂